ehrql/tables/beta/tpp.py

```diff
@@ -237,6 +238,40 @@ class household_memberships_2020(PatientFrame):
     household_size = Series(int)
 
 
+@table
+class ons_cis_raw(EventFrame):
+    """
+    Data from the ONS Covid Infection Survey.
```

Comment:
Is it worth clarifying how this table differs from `ons_cis` below? As it stands they have exactly the same description.

Reply:
Yes, it is 🙂 I've clarified and force-pushed.